tests/attack/test_mod_sql.py

```diff
@@ -103,7 +104,7 @@ async def test_true_positive():
 
         assert persister.add_payload.call_count
         assert persister.add_payload.call_args_list[0][1]["module"] == "sql"
-        assert persister.add_payload.call_args_list[0][1]["category"] == "SQL Injection (DBMS: MySQL)"
+        assert persister.add_payload.call_args_list[0][1]["category"] == SqlInjectionFinding.name()
```

Comment:
You should keep static values instead of functions in test files to really make sure it produces the expected result

Reply:
Put back strings